languages/translations/ru.json

```diff
@@ -62,49 +62,57 @@
                 "default": "Вы прибыли в {nth} пункт назначения",
                 "upcoming": "Вы прибудете в {nth} пункт назначения",
                 "short": "Вы прибыли",
-                "short-upcoming": "Вы скоро прибудете"
+                "short-upcoming": "Вы скоро прибудете",
+                "named": "Вы прибыли в {waypoint_name}"
```

Comment:
It's better to change this text and below to _"Вы прибыли в пункт назначения, {waypoint_name}"_. Otherwise currently used form requires prepositional case applying to `{waypoint_name}` value.

And maybe it's better to publish new strings templates in Transifex first?

Reply:
Thanks @yuryleb, does that mean the grammar rules applied [on this line](https://github.com/Project-OSRM/osrm-text-instructions/blob/master/index.js#L254) will not take care of it for us?

Reply:
> And maybe it's better to publish new strings templates in Transifex first?

One of the unfortunate aspects of OSRMTI currently is that adding a new string requires immediately adding _something_ in each of the languages, whether that’s a copy of the English string or something hacked together based on preexisting translations. Transifex can only pick up new strings once the initial PR lands.

We could just insert English everywhere as a placeholder, but that would result in a temporary regression in all other languages. Regressing translation coverage isn’t necessarily a big deal, but some languages have taken months to update, even if it’s just a matter of adapting an existing string as this PR does. #84 and #155 track figuring out a better process for adding new strings.